atlas/templates/cmd/server/grpc.go.gotmpl

```diff
@@ -62,3 +64,10 @@ func NewGRPCServer(logger *logrus.Logger{{ if or .WithPublishTopic .WithSubscrib
 
 	return grpcServer, nil
 }
+
+{{ if .WithDatabase }}
+// setMaxOpenIdleConnection sets the db connection string
+func setMaxOpenIdleConnection(db *gorm.DB) {
```

Comment:
I might call it something like `configureDBConns()`.

It would be nice to set the [`SetConnMaxIdleTime`](https://pkg.go.dev/database/sql#DB.SetConnMaxIdleTime) and [`SetConnMaxLifetime`](https://pkg.go.dev/database/sql#DB.SetConnMaxLifetime) as well.

Reply:
sure